backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/util/TimeUtil.java

```diff
@@ -2230,4 +2230,40 @@ public static long getMillisBetween(@NonNull final Timestamp timestamp1, @NonNul
 	{
 		return timestamp2.getTime() - timestamp1.getTime();
 	}
+
+	@Nullable
+	public static Instant asStartOfDayInstant(@Nullable final LocalDate localDate, @NonNull final ZoneId zoneId)
+	{
+		if (localDate == null)
+		{
+			return null;
+		}
+		final LocalDateTime endOfDay = localDate.atTime(LocalTime.MIN);
```

Comment:
@fp-ps shouldn't be LocalTime.MAX?

Reply:
ahhh.... the `endOfDay` variable shall be renamed, i guess :)

Reply:
yees, I renamed it